packages/usa-input-prefix-suffix/src/styles/_usa-input-prefix-suffix.scss

```
@@ -13,6 +13,11 @@
     @include focus-outline();
   }
 
+  &--disabled {
+    background-color: color("disabled");
+    cursor: not-allowed;
```

Should this use the mixin? 
```suggestion
    @include u-disabled;
```

---

Resolved in [12c3713](https://github.com/uswds/uswds/pull/5063/commits/12c371351d9459d2c23509ee84a10068ba74e675)